src/H5Opublic.h

```diff
@@ -2018,13 +2022,12 @@ H5_DLL hid_t  H5Oopen_by_addr(hid_t loc_id, haddr_t addr);
  *       will force metadata entries to be evicted from the cache,
  *       thus freeing the memory associated with the entries.
  *
- * \version 1.10.5 The macro #H5Oget_info was removed and the function H5Oget_info1() was copied to H5Oget_info().
- * \version 1.10.3 Function H5Oget_info() was copied to H5Oget_info1(), and the macro #H5Oget_info was created.
- * \version 1.8.15 Added a note about the valid values for the \c version field in the H5O_hdr_info_t structure.
- * \version 1.8.11 Fortran subroutine introduced in this release.
- * \version 1.8.10 Added #H5O_type_t structure to the Description section. \n
- *                 Separated H5O_hdr_info_t structure from #H5O_info_t in the Description section. \n
- *                 Clarified the definition and implementation of the time fields.
+ * \version 1.10.5 The macro #H5Oget_info was removed and the function H5Oget_info1() was copied to
```

Comment:
The changes on these 7 lines look undesirable to me.  Should the original formatting be preserved?

Reply:
I generally agree with you, but that seems like something for the Doxygen team to pick up and not part of this PR.  (Since I only ran the source formatting script on the code, with no other changes from me)